src/common/hooks/useVersionHistoryFormatters/useVersionHistoryFormatters.test.js

```diff
@@ -0,0 +1,89 @@
+/* Developed collaboratively using AI (Chat GPT) */
+
+import { renderHook } from '@folio/jest-config-stripes/testing-library/react';
```

Comment:
Please get acquainted with code style guides that are not covered with ES Lint, particularly with import styles https://folio-org.atlassian.net/wiki/spaces/FOLIJET/pages/1377187/Code+style+rules+not+covered+by+eslint

Reply:
No. The correct order of imports is 
1. from third-parties, e.g. `from 'foo'`
2. from FOLIO, e.g. `from '@folio/...'`
3. from local paths, e.g. `from './foo'`